test/TEST-03-JOBS/test-jobs.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -ex
+#!/usr/bin/env bash -ex
```

Comment:
There are more of such invocations. Please look at the other test scripts.

Reply:
Now I covered them all.